tests/test_bookmarks.py

```diff
@@ -18,10 +18,14 @@ def name(self):
     def testable_streams(self):
         return self.expected_streams().difference(
             {  # STREAMS NOT CURRENTY TESTABLE
-                'employees',
+                'employees', # Requires production environment to create records
                 'locations'
             }
         )
+    @classmethod
+    def tearDownClass(cls):
+        print("\n\nTEST TEARDOWN\n\n")
+        # TODO add delete for locations if possible
```

Comment:
Can we just add the teardown later when we need it?

Reply:
We can, I like having the print statement because it helps separate the test output in circle. But it definitely isn't needed.

Reply:
Ok fair enough that's fine.